include/core/Partition_One.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef PARTITION_ONE_H
+#define PARTITION_ONE_H
+
+#ifdef NGEN_MPI_ACTIVE
+
+#include <iostream>
+#include <memory>
+#include <string>
+#include <vector>
+#include <unordered_set>
+#include <FeatureBuilder.hpp>
+#include "features/Features.hpp"
+#include <FeatureCollection.hpp>
+#include "Partition_Data.hpp"
+
+class Partition_One {
+
+    public:
+        Partition_One() {};
+
```

Comment:
```suggestion
```
The compiler will provide a default constructor without any need to write something out. Specifying a non-default constructor can create unintended side effects.